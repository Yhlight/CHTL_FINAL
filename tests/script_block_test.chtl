html {
    head {
        title { text: "Script Block Test"; }
    }
    body {
        div {
            id: "my-div";
            text: "Hello, World!";
        }

        script {
            {{#my-div}}.style.color = "red";
        }
    }
}
