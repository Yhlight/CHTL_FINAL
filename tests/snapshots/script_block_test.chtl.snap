<html>
  <head>
    <title>
      Script Block Test
    </title>
  </head>
  <body>
    <div id="my-div">
      Hello, World!
    </div>
  </body>
</html>
